TESTING.md

```diff
@@ -0,0 +1,41 @@
+# How to test
+
+## Install the new Promtail version
+
+* Choose a test cluster: `opsctl login gauss`
+* Suspend these 2 flux kustomizations:`flux suspend kustomization -n flux-giantswarm collection; flux suspend kustomization -n flux-giantswarm flux`
+
+### The "easy" way
+
+* Deploy the new Promtail version using opsctl (replace `test-branch` with your branch's name): `opsctl deploy --use-kubeconfig promtail-app@test-branch`
+
+### The "hard" way
+
+If for some reason you can't use the `opsctl deploy` command or don't want to:
+* Either create a draft PR or directly use your branch
```

Comment:
Does the "easy" way work without that? Easy way also relies on the apps pushed to the test catalogs, and AFAIK this is done on PR. So I'd say this line goes before chosing between easy/hard.

Reply:
True, moving this